app/models/item.rb

```diff
@@ -0,0 +1,6 @@
+class Item < ActiveRecord::Base
+  attr_accessible :name, :price
+  belongs_to :meal, :presence => true
```

Comment:
For readability, I think it's better if we split up validations from relationship declaration, especially when we do more complicated validations. In other words, instead of `belongs_to :meal, :presence => true`, it might be nicer to do:
```ruby
belongs_to :meal
validates :meal, :presence => true
```